src/rules/media-feature-value-dollar-variable/index.js

```diff
@@ -53,6 +55,11 @@ export default function(expectation) {
           });
         }
 
+        // A keyword value, like `none`, should always be fine
+        if (valueParsed.search(keywordValueRegex) !== -1) {
```

Comment:
Maybe use `test` since you only check for a boolean here?

Reply:
Thanks, update!